migrate/index.md

```diff
@@ -55,3 +70,4 @@ If you encounter any difficulties while migrating, consult the
 [pg-dump-restore]: /migrate/:currentVersion:/pg-dump-and-restore/
 [parallel-copy]: /use-timescale/:currentVersion:/ingest-data/import-csv/
 [troubleshooting]: /migrate/:currentVersion:/troubleshooting/
+[live-migrations]: /migrate/:currentVersion:/live-migrations/
```

Comment:
Note to self, before merging verifying this path is aligned with @alejandrodnm PR.